docs/01-app/03-building-your-application/03-rendering/05-edge-and-nodejs-runtimes.mdx

```diff
@@ -14,6 +14,8 @@ Next.js has two server runtimes you can use in your application:
 - The **Node.js Runtime** (default), which has access to all Node.js APIs and compatible packages from the ecosystem.
 - The **Edge Runtime** which contains a more limited [set of APIs](/docs/app/api-reference/edge).
 
+The Edge Runtime is the default runtime for [Middleware](/docs/app/building-your-application/routing/middleware). However, this can be changed to the experimental Node.js runtime. See the [middleware documentation](/docs/app/building-your-application/routing/middleware#using-nodejs-runtime-with-middleware) for more details.
```

Comment:
```suggestion
The Edge Runtime is the default runtime for [Middleware](/docs/app/building-your-application/routing/middleware). However, this can be changed to the Node.js runtime. See the [Middleware documentation](/docs/app/building-your-application/routing/middleware#using-nodejs-runtime-with-middleware) for more details.
```